packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -222,28 +224,38 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   Widget _expandCollapseButtons() {
     if (!_expandCollapseSupported) return null;
 
+    const minIncludeTextWidth = 900.0;
+    final buttonStyle = denseAwareOutlinedButtonStyle(
+      context,
+      minIncludeTextWidth,
+    );
+
     return Align(
       alignment: Alignment.centerRight,
       child: Row(
         mainAxisAlignment: MainAxisAlignment.end,
         mainAxisSize: MainAxisSize.min,
         children: [
-          Flexible(
+          SizedBox(
             child: OutlinedButton(
+              style: buttonStyle,
               onPressed: enableButtons ? _onExpandClick : null,
-              child: const Text(
+              child: const ImageIconLabel(
+                Icon(Icons.unfold_more),
                 'Expand all',
-                overflow: TextOverflow.ellipsis,
+                minIncludeTextWidth: minIncludeTextWidth,
               ),
             ),
           ),
-          const SizedBox(width: denseSpacing),
-          Flexible(
+          SizedBox(width: isDense() ? denseModeDenseSpacing : denseSpacing),
```

Comment:
revert this padding back to dense spacing. We can add a line to IconLabelButton that will help with the width problem.